nextcloud-aio-helm-chart/readme.md

```diff
@@ -12,3 +25,13 @@ Then run:
 helm repo add nextcloud-aio https://nextcloud.github.io/all-in-one/
 helm install my-release nextcloud-aio/nextcloud-aio-helm-chart -f values.yaml
 ```
+
+And after a while, everything should be set up.
+
+## How to update?
+Since the values of this helm chart may change in the future, it is highly recommended to strictly follow the following procedure whenever you want to upgrade it.
+1. Stop all running pods
+1. Back up all volumes that got created by the Helm chart and the values.yaml file
+1. Run `helm repo update nextcloud-aio` in order to get the updated yaml files from the repository
+1. Now download the updated values.yaml file from https://raw.githubusercontent.com/nextcloud/all-in-one/main/nextcloud-aio-helm-chart/values.yaml and compare that with the one that you currently have locally. Look for variables that changed or got added. You can use the diff command to compare them.
```

Comment:
It might be a good idea to link to the latest released values.yaml (if that is possible). I could imagine someone installing or updating to a newer release and accidentally downloading a in-development version with changed variables.

Is it possible to statically link to a file in the tree of the latest release/tag?

Reply:
Actually since the Helm update script will trigger directly a new release, getting this file should always match the latest version of the helm chart...